gdnative-core/src/nativescript/init.rs

```diff
@@ -96,6 +96,9 @@ impl InitHandle {
     where
         C: NativeClassMethods,
     {
+        if !register_class::<C>() {
+            panic!("NativeClass type `{type_name}` has already been added to the `InitHandle`; please find duplicate `handle.add_class::<{type_name}>` entry in function called by `godot_nativescript_init` macro.", type_name = std::any::type_name::<C>());
```

Comment:
This message might be a bit too long imo. It should be sufficient to point out that `` `{type_name}` has already been registered ``: if you know how to register a type, then you already know how you can find a duplicate.